src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/ElytraImpactListener.java

```diff
@@ -18,51 +22,63 @@
 import io.github.thebusybiscuit.slimefun4.implementation.Slimefun;
 import io.github.thebusybiscuit.slimefun4.implementation.items.armor.ElytraCap;
 import io.github.thebusybiscuit.slimefun4.implementation.items.armor.SlimefunArmorPiece;
+import org.bukkit.event.entity.EntityToggleGlideEvent;
 
 /**
  * The {@link Listener} for the {@link ElytraCap}.
  *
  * @author Seggan
+ * @author J3fftw1
  * 
  * @see ElytraCap
  */
 public class ElytraImpactListener implements Listener {
 
+    private final Set<UUID> gliding =  new HashSet<>();
+
     public ElytraImpactListener(@Nonnull Slimefun plugin) {
         plugin.getServer().getPluginManager().registerEvents(this, plugin);
     }
 
+    @EventHandler
+    public void onGlideToggle(EntityToggleGlideEvent event) {
+        Entity entity = event.getEntity();
+        if (entity instanceof Player player && player.isGliding()) {
+            UUID uuid = player.getUniqueId();
+            gliding.add(uuid);
+        }
+        Slimefun.instance().getServer().getScheduler().runTaskLater(Slimefun.instance(), gliding::clear, 1);
```

Comment:
Can we just have some docs/comments somewhere detailing why we add them/clear them 1 tick later from the group? We know its required and we know why but just reading the code its reads a bit werid not knowing about the 'forced toggle' minecraft does

Reply:
Not sure if i made the right comment i cant exactly remember why we do it